source/opt/cfg.cpp

```diff
@@ -111,5 +124,18 @@ void CFG::ComputeStructuredSuccessors(ir::Function* func) {
   }
 }
 
+void CFG::ComputePostOrderTraversal(BasicBlock* bb,
+                                    std::vector<BasicBlock*>& order,
```

Comment:
Syle guide says all references must be const [here](https://google.github.io/styleguide/cppguide.html#Reference_Arguments).

Reply:
Yes, that is correct.  Fixed.